DZ_Shabalina1.xml

```diff
@@ -0,0 +1,12 @@
+<Primary_School_Student>
```

Comment:
Not a mistake, but if we want to highlight it's a student of a primary school we can call the main object

```xml
<student>
     <type_of_school>Primary</type_of_school>
</student>
```
Just as an option